### DISCUSSION TITLE
Context is not available when using yarn run generate

### DISCUSSION THREAD

**Discussion by imjohnlouie04** [nuxt/nuxt#17088] [General]
<img width="512" alt="Screen Shot 2022-08-12 at 9 35 17 PM" src="https://user-images.githubusercontent.com/44358556/184364854-647fb940-7398-4c1c-8943-9e1020747bb2.png">

**Comment by kissu** [comment]
Hi, mind giving more info or a repro please? 😁

**Reply by nestle49** [reply]
The same error after migrate on nuxt/bridge after npm run dev
issue: https://github.com/nuxt/bridge/issues/474
